tab_err/error_mechanism/_enar.py

```diff
@@ -31,10 +57,10 @@ def _sample(self: ENAR, data: pd.DataFrame, column: str | int, error_rate: float
             msg += f"However, only {len(se_data_error_free)} error-free cells are available."
             raise ValueError(msg)
 
+        # To-Do? -- ensure that the implementation is consistent between the ear and enar implementations of _sample -- upper_bound_variable?
         lower_error_index = self._random_generator.integers(0, len(se_data_error_free) - n_errors) if len(se_data_error_free) != n_errors else 0
-
         error_index_range = range(lower_error_index, lower_error_index + n_errors)
-        selected_rows = se_data_error_free.sort_values().iloc[error_index_range]
+        selected_rows = se_data_error_free.sort_values().iloc[error_index_range]  # Introduce errors to locations of sorted values
```

Comment:
```suggestion
        selected_rows = se_data_error_free.sort_values().iloc[error_index_range]
```

I think that's clear enough without this comment.